src/ui/public/partials/saved_object_finder.html

```diff
@@ -29,19 +52,26 @@
       ng-keydown="finder.hitKeyDown($event, page, paginate)"
       ng-click="finder.onChoose(hit, $event)">
 
-      <a
-        ng-href="{{finder.makeUrl(hit)}}"
-        ng-blur="finder.hitBlur($event)"
-        ng-click="finder.preventClick($event)">
-
-        <i aria-hidden="true" class="fa" ng-if="hit.icon" ng-class="hit.icon"></i> {{hit.title}}
-        <p ng-if="hit.description" ng-bind="hit.description"></p>
+      <a ng-href="{{finder.makeUrl(hit)}}"
+          ng-blur="finder.hitBlur($event)"
+          ng-click="finder.preventClick($event)">
+        <ul>
+          <li ng-show="finder.properties.noun === 'Visualization'" class="finder-type">
```

Comment:
Same here, why not just look to see if there's a hit.icon on the hit like the row below?